src/nvim/version.c

```diff
@@ -197,7 +197,7 @@ static int included_patches[] = {
   // 1096,
   // 1095 NA
   // 1094,
-  // 1093,
+  // 1093 NA
```

Comment:
Why is this NA? `f_cursor()`in eval.c is also modified (error message, function comment).